tests/pmap_test.py

```diff
@@ -494,6 +494,29 @@ def sum_helper(a):
     ans = f(x)
     self.assertAllClose(ans, expected, check_dtypes=False)
 
+  def testGatherReplicaGroups(self):
+    replicas = xla_bridge.device_count()
+    if replicas % 2 != 0:
+      raise SkipTest
```

Comment:
It would be nice to add a message for `SkipTest`. I see that some tests in this file don't do that, but it is still a good thing to do.

Reply:
Done.